src/main/java/gregtech/api/block/VariantActiveBlock.java

```diff
@@ -58,9 +59,15 @@ protected boolean canSilkHarvest() {
         return false;
     }
 
+    @Override
+    public BlockRenderLayer getRenderLayer() {
+        return BlockRenderLayer.CUTOUT;
+    }
+
     @Override
     public boolean canRenderInLayer(IBlockState state, BlockRenderLayer layer) {
-        return layer == BlockRenderLayer.CUTOUT || layer == BloomEffectUtil.getRealBloomLayer();
+        return layer == getRenderLayer() ||
+                layer == (isBloomEnabled(getState(state)) ? BloomEffectUtil.getRealBloomLayer() : BlockRenderLayer.CUTOUT);
```

Comment:
Instead of hard-coding CUTOUT here, I think it would be better to do something like this:

```suggestion
        return layer == getRenderLayer() ||
                (isBloomEnabled(getState(state)) && layer == BloomEffectUtil.getRealBloomLayer());
```

Reply:
The CUTOUT check is for rendering bloom textures when the bloom config is turned off. The alternative suggested is functionally different, as it could potentially prevent drawing on CUTOUT layer.